tests/test_bukuDb.py

```diff
@@ -793,7 +793,7 @@ def test_close_quit(self):
     # self.fail()
 
 
-@pytest.mark.parametrize('status', [None, 200, 302, 307, 404, 500])
+@pytest.mark.parametrize('status', [None, 200, 302, 308, 404, 500])
```

Comment:
…I figured that at least _one_ **permanent** redirect HTTP code is necessary to test how `add_rec()` handles it :sweat_smile: